service/src/main/java/eu/europa/ec/fisheries/ers/activity/message/consumer/bean/ActivityMessageConsumerBean.java

```diff
@@ -99,11 +106,13 @@ public void onMessage(Message message) {
             }
             switch (method) {
                 case GET_FLUX_FA_REPORT:
```

Comment:
It looks weird how we send "saveReportRequest" to "handleFaReportSaving", under the case labeled "GET_FLUX_FA_REPORT". Are we getting or are we saving?

Reply:
It's saving a report but it's definitely a weird name, but can't really change the method types right now.